src/hyperlight_host/src/hypervisor/gdb/mod.rs

```diff
@@ -287,167 +274,6 @@ pub(crate) enum DebugResponse {
     WriteRegisters,
 }
 
```

Comment:
couldn't we keep some of this trait here instead of moving it up into the new vm trait?

Reply:
We could, I could make it a supertrait of vm

Reply:
fixed